internal/auditlog/https_writer.go

```diff
@@ -20,9 +20,10 @@ import (
 // httpsWriter is used to store logs in a single file
 type httpsWriter struct {
 	io.Closer
-	formatter plugintypes.AuditLogFormatter
-	url       string
-	client    *http.Client
+	formatter   plugintypes.AuditLogFormatter
+	url         string
+	client      *http.Client
+	contentType string
```

Comment:
I think content type should be on the formatter, not https writer. Since it's still experimental API, we can change formatter, how about making it an actual type with `Format()` and `ContentType()` methods?

Reply:
Brilliant. Let's do it.